packages/zmarkdown/__tests__/__snapshots__/latex.tests.js.snap

```diff
@@ -85,17 +85,19 @@ a code without lang
 `;
 
 exports[`code+caption 1`] = `
-"\\\\begin{codeBlock}[With Source]{python}
+"\\\\begin{CodeBlock}{python}
 print('bla')
-\\\\end{codeBlock}
+\\\\captionof{listing}{With Source}
+\\\\end{CodeBlock}
 
 
 
-\\\\begin{codeBlock}[With Source][1,2]{python}
+\\\\begin{CodeBlock}[1,2]{python}
```

Comment:
.

Reply:
ping @pierre-24 ?

Reply:
Will not work, but `\\\\begin{CodeBlock}[][1,2]{python}` will.

Reply:
Fixed